src/test/java/weblogic/logging/exporter/ResultTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+ */
+
+package weblogic.logging.exporter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Test the Result object")
+public class ResultTest {
+
+  private static final String EXPECTED_STRING =
+      "Result{response='good', status=1, successful=true}";
+
+  @DisplayName("Check constructor and getters")
+  @Test
+  public void testConstructorAndGetters() {
```

Comment:
I have found that testing style makes a big difference. You want your test names to explain what functionality you expect. A test name like "testConstructorAndGetters" provides essentially no information. In this case, I'd be inclined to name the test something like: "afterCreated_canRetrieveFieldValues".

Also, hard-coded numbers are often less than helpful. What is '1'? The number appears to be an HTTP status, so it might be more information to use an HTTP status constant defined in whatever library this is used for.

(I'm not crazy about the Result class at all, but you're just writing the tests that didn't get written, so that's not relevant).